src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -38,7 +38,8 @@
         },
         "VLAN_INTERFACE": {
             "Vlan111": {
-                "nat_zone": "0"
+                "nat_zone": "0",
+                "static-anycast-gateway": "true"
```

Comment:
Just to be consistent, please use"_" instead of "-" i.e static_anycast_gateway : "true"?

Reply:
Got it, will update it.